extension/linkify-urls-in-code.js

```diff
@@ -14,7 +14,7 @@ window.linkifyURLsInCode = (() => {
 	const linkifyURL = url => `<a href="${url}" ${commonURLAttrs}>${url}</a>`;
 
 	const hasIssue = text => issueRegex.test(text);
-	const hasURL = text => URLRegex.test(text);
+	const findURLs = text => text.match(URLRegex) || [];
```

Comment:
What name would be better? `findURLs` or `matchURLs`? After taking another look at the PR it would literally describe what the function is doing. WDYT?